.github/workflows/swift-toolchain.yml

```diff
@@ -119,10 +119,10 @@ jobs:
       yams_revision: ${{ steps.context.outputs.yams_revision }}
       zlib_revision: ${{ steps.context.outputs.zlib_revision }}
       ANDROID_API_LEVEL: ${{ steps.context.outputs.ANDROID_API_LEVEL }}
-      WINDOWS_CMAKE_C_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_C_FLAGS }}
-      WINDOWS_CMAKE_CXX_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_CXX_FLAGS }}
-      WINDOWS_CMAKE_EXE_LINKER_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_EXE_LINKER_FLAGS }}
-      WINDOWS_CMAKE_SHARED_LINKER_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_SHARED_LINKER_FLAGS }}
+      HOST_CMAKE_C_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_C_FLAGS }}
+      HOST_CMAKE_CXX_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_CXX_FLAGS }}
+      HOST_CMAKE_EXE_LINKER_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_EXE_LINKER_FLAGS }}
+      HOST_CMAKE_SHARED_LINKER_FLAGS: ${{ steps.context.outputs.WINDOWS_CMAKE_SHARED_LINKER_FLAGS }}
```

Comment:
These really should be Windows - they contain Windows specific flags, which is why we have the `ANDROID_CMAKE_C_FLAGS` as well. If you want to collapse these somehow then we should only have `CMAKE_C_FLAGS`. We also have more than one host that we build for, so `HOST_` is a poor choice of prefix.

Reply:
This will make a bit more sense once #793 has landed but it is stalled on fixing the build. The idea here is that these are going to be referring to flags used for the host platform (either Mac or Windows). When sub-workflow is set up, `HOST_CMAKE_*_FLAGS` will be set by the main workflow to either `WINDOWS_CMAKE_*_FLAGS` or `DARWIN_CMAKE_*_FLAGS`, depending on the build.

Reply:
The problem is that there is no one canonical host. We have the host for the toolchain and the host for the runtimes which are different. Technically, it would be host and target, but we don't call the target flags the target flags but name them according to the target.

Reply:
Ack. I am abandoning that part of the change and will make a separate change to split the usage.